server/models/meteorological_datum.py

```diff
@@ -0,0 +1,33 @@
+from sqlalchemy import Column, DateTime, Float, ForeignKey, Integer, String
+
+from server.db.base_class import Base
+from server.models.buoy import Buoy
+
+
+class MeteorologicalDatum(Base):
+    __tablename__ = "meteorological_data"
+
+    id = Column(Integer, index=True, nullable=False, primary_key=True)
+    station_id = Column(
+        String(10),
+        ForeignKey(Buoy.station_id),
+        index=True,
+        nullable=False,
+        unique=False,
+    )
+    date_recorded = Column(DateTime, nullable=False)
+    wind_direction = Column(Integer, nullable=True)
+    wind_speed = Column(Float, nullable=True)
+    wind_gust = Column(Float, nullable=True)
+    wave_height = Column(Float, nullable=True)
+    dominant_period = Column(Integer, nullable=True)
+    average_wave_period = Column(Float, nullable=True)
+    mean_wave_direction = Column(String, nullable=True)
```

Comment:
Rename the following to align with both NDBC's descriptions and `pybuoy`, which conforms to it:

- mean_wave_direction to wave_direction
- dewpoint to dewpoint_temperature

And remove salinity here. I don't think that's captured for meteorological. 

Doing so will make life easier when loading parsed data from `pybuoy` to our database.